### DISCUSSION TITLE
Question on the AzureOpenAI translation (404 not found)

### DISCUSSION THREAD

**Discussion by damon98xp** [dotnet/ResXResourceManager#644] [Q&A]
Hi team,

I encountered below error using Azure Open AI translation service, could you please enlighten how to solve this issue? (I'm sure the API key is working), Thanks.

![image](https://github.com/dotnet/ResXResourceManager/assets/1480581/93ffbfc3-60e1-4c13-a4b1-f1eeadeae20c)

**Comment by tom-englert** [comment]
Are you sure `xxxtranslator` is the correct url?

**Comment by damon98xp** [comment]
Correct endpoint is https://wttranslator.openai.azure.com/

![image](https://github.com/dotnet/ResXResourceManager/assets/1480581/d8b10b66-73f5-497f-a627-5683c0293808)


________________________________
寄件者: tom-englert ***@***.***>
寄件日期: Monday, June 10, 2024 9:31:37 PM
收件者: dotnet/ResXResourceManager ***@***.***>
副本: Damon98xp ***@***.***>; Author ***@***.***>
主旨: Re: [dotnet/ResXResourceManager] Question on the AzureOpenAI translation (404 not found) (Discussion #644)


Are you sure xxxtranslator is the correct url?
